app/lambda/custom_resource_athena/queries/createNamedQueries.json

```diff
@@ -0,0 +1,29 @@
+{
+    "queries" : [
+        {
+            "name": "custom query 1",
```

Comment:
Can we have some meaningful names for the queries? It could be the short version of description too.

Reply:
Done. https://github.com/aws-samples/medical-transcription-analysis/blob/f05b763e09b17a207c02070a30bf2ee564f1d99d/app/lambda/custom_resource_athena/queries/createNamedQueries.json#L3-L7